src/fides/core/export.py

```diff
@@ -229,10 +229,12 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
                 keys.append(key_string)
                 output_list[0] = tuple(keys)
                 custom_columns[key_string] = key
+                child_custom_field_data = {}
                 if isinstance(value, list):
-                    system_custom_field_data[key_string] = ", ".join(value)
+                    child_custom_field_data[key_string] = ", ".join(value)
```

Comment:
some poor naming potentially, but I ended up filtering based on the system `fides_key` and adding the custom fields as children of said `fides_key`